### ISSUE TITLE
Addition to the Delivery job

### ISSUE THREAD

**Issue by Goten87** [ldtteam/minecolonies#1270]
### Minecolonies version

version 1.11.2-0.8.3061

### New system to be made

i was playing a little with the mod and i like it so far
what i will like to see is a chest (your own one) where we as the player can drop items for the delivery man/woman to pick up and bring to the warehouse and sort it
also a chest where we can say keep items in it so if 1 item is it it the delevery npc will always keeps a stack in it if its a tool it will always keeps 2 or more items in it

also will a delivery man/woman pickup items from AE2 for example the chest or the terminal

**Comment by Asherslab** [comment]
These things will be a lot more do-able once the Request System is done, there have been several suggestions for AE2 so I'd suggest looking at the responses to those =D

**Comment by Raycoms** [comment]
Parts of the request system and separate pr
